### DISCUSSION TITLE
Estimated Amount Clarification

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#155] [Q&A] [answered]
On the latest CMS presentation documentation, for Estimated Allowed Amount it mentions that the requirement includes "hybrid scenarios where the standard charge dollar is a base rate and there is an algorithm that accounts for additional individualized charges".  What exactly is that referring to?
If a Contract has a Stop Loss clause for all Inpatient services and the DRG reimbursement is a Case Rate only. Is an estimate required?

**Comment by RidenourK** [comment, marked as answer]
@JaSimps5. Thank you for your question.

As indicated in the CY 2024 OPPS/ASC Final Rule, hospitals are required to display the standard charges as they are established, such that, if the hospital established a standard charge as a dollar amount, the hospital would display the standard charge as a dollar amount. If the hospital has established a standard charge as a percentage or algorithm such that a standard dollar amount is not available, then the hospital would display the standard charge as a percentage or algorithm… We anticipate that most if not all payer-specific negotiated charges will fall into one of three categories, depending on how a hospital has established them: (1) standard dollar amount, (2) standard algorithm or percentage, or (3) hybrid where a standard dollar amount can be identified but the final allowed amount is dependent on additional variables. An example of where we would expect to see a standard charge in dollars would be standard charges established under a fee schedule or where an identifiable dollar amount has been established for an item or service (for example, a per diem rate, a gross charge for an itemized item or service, or a cash discounted price for a service package). An example of a where we would expect to see a standard charge expressed as an algorithm would be when a hospital has negotiated a reimbursement for defined service packages (for example, hip replacement or colonoscopy) that are based on differential percentages of total billed (gross) charges (for example, 50 percent of total billed charges for hip replacement and 75 percent of total billed charges for colonoscopy). A hybrid would be a situation in which the hospital has established both a standard charge in dollars and there are additional variables that would modify the negotiated rate for a particular item or service. For example, a hospital may have established a payer-specific negotiated charge under the MS-DRG methodology where an adjusted base rate in dollars has been established for each DRG code, but the adjusted base rate may be further modified due to certain variable factors (such as outlier cases or transfers). In general, we recommend that each hospital, as a starting point, inspect their contracts with each third-party payer to identify all standard charges established as dollar amounts. Next, the hospital should populate, by payer and plan, the MRF with those standard charges in dollars and describe the item or service associated with each of the standard charges (along with any relevant billing or accounting codes). After that has been done, the hospital should identify whether the standard charge (in dollars) is subject to modification and what factors or variables (for example, algorithm) might cause the standard charge to change, and indicate those as instructed by the data dictionary for the particular format selected. If the hospital's payer-specific negotiated charge is based on an algorithm within which no standard dollar amount can be determined, then the hospital should specify what percentage or algorithm determines the dollar amount for the item or service. Please refer to [88 FR 82097](https://www.federalregister.gov/d/2023-24293/p-4970).
